### PR TITLE
VACMS-4081: Remove nickname field from config.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
See #4081 - second step in field removal process. First step (https://github.com/department-of-veterans-affairs/vets-website/pull/15791 = merged) removed graphql / transformer instances from vets-web.

## Testing done
Behat

## Screenshots
N/A

## QA steps
N/A

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Core Application Team`
- [x] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

**Comment by swirtSJW** [comment]
I am holding on this for 2 reasons:
1. This won't pass our tests until we update what vetsweb version we are pulling.  (you'll need to update vetwsweb in our repo to do this)
2.  THere is timing confusion that makes no sense to me on vetsweb.  I suspect tagging the release is not contingent on it successfully going out.   
image

File: composer.lock [hunk 1]
```diff
@@ -18953,16 +18938,16 @@
         },
         {
             "name": "va-gov/web",
-            "version": "v0.1.1242",
+            "version": "v0.1.1248",
```
Comment (swirtSJW) [inline composer.lock:18941]: This is the tag that contains your code update `v0.1.1257`  It needs to be updated to that release.

**Review by swirtSJW — CHANGES_REQUESTED** [review]
Incorrect vetweb version pulled in.

**Comment by swirtSJW** [comment]
I also think we may need to include ripping them out of transformers too.   Otherwise we will delay the transformer team that is already stressed enough.

**Comment by ethanteague** [comment]
Apologies, @swirtSJW - I have this in draft state / didn’t mean for you to review it yet. I’ll pull in the correct vets web version tomorrow. My front end pr that was merged today pulled out the field from vets web transformer logic.

**Review by swirtSJW — APPROVED** [review]

**Comment by swirtSJW** [comment]
image

**Pull request merged**